pkg/k8sutil/constants.go

```diff
@@ -17,7 +17,7 @@ package k8sutil
 const (
 	// KubeConfigEnvVar defines the env variable KUBERNETES_CONFIG which
```

Comment:
Actually the comment needs to be updated as well. Change `KUBERNETES_CONFIG` to `KUBECONFIG`.